less/index.less

```diff
@@ -57,3 +57,76 @@ html, body {
     height: 100%;
 }
 
+
+// ================= General Styles:=========
+
+html {
+	font-size: 62.5%;
+	}
+
+.container {
+		max-width: 880px;
+		margin: 0 auto;
+		border: 1px solid red;
+		font-size: 1.67rem;
+		padding: 20px 40px;
+	}
```

Comment:
One of your objectives is to nest all your selectors inside of your main container